src/validation-group.js

```diff
@@ -491,7 +491,7 @@ export class ValidationGroup {
    * @param message either a static string or a function that takes two arguments: newValue (the value that has been evaluated) and threshold.
    * @returns {ValidationGroup} returns this ValidationGroup, to enable fluent API
    */
-  withMessage(message) {
+  withMessage(message: any): ValidationGroup {
```

Comment:
Sorry, I mainly updated more what I was more confident about. Updated as you suggested.